crypto/keyring/keyring_test.go

```diff
@@ -175,7 +175,7 @@ func TestSignVerifyKeyRing(t *testing.T) {
 
 	// try signing both data with both ..
 	s11, pub1, err := kb.Sign(n1, d1)
-	require.NoError(t, err)
+	require.NoError(t, err) //TODO fix no registered implementations of type types.PrivKey
```

Comment:
I get an error `no registered implementations of type types.PrivKey` while running [UnmarshalInterface](https://github.com/cosmos/cosmos-sdk/blob/kerying-migration/crypto/keyring/record.go#L189). I guess, `types.PrivKey` has to be registered in codec(codec.Codec) implementations like it is done in [Amino](https://github.com/cosmos/cosmos-sdk/blob/kerying-migration/crypto/keyring/codec.go#L16-L23). Where I can do it?

Reply:
In crypto/codec/proto.go

Reply:
As @amaurym  noted, probably the easiest is to add:
```
	var sk *cryptotypes.PrivKey
	registry.RegisterInterface("cosmos.crypto.PrivKey", sk)
	registry.RegisterImplementations(sk, &ed25519.PrivKey{})
	// other kind of keys...
```
to the `RegisterInterfaces` function.